<launch>
    <arg name="sensors_topics" default="['cam_poses', 'lidar_poses']"/>

    <!-- Node to run the pose_fusion_node -->
    <node pkg="mercator_sensor_fusion_ros_pkg" type="pose_fusion" name="pose_fusion" output="screen">
        <!-- Parameters for the node -->
        <rosparam param="sensors_topics" subst_value="True">$(arg sensors_topics)</rosparam>
        <param name="sensors_trusts" value="[true, false]"/>
        <param name="keep_tracking_with_only_lidar" value="false"/>
        <param name="sensors_number" value="2"/>
        <param name="process_noise_factor" value="0.0001"/>
        <param name="frequency" value="25.0"/>
        <param name="initial_sensors_variances" value="[0.0256, 0.0196]"/>
        <param name="robot_average_radius" value="0.15"/>
        <param name="initial_velocity" value="0.0"/>
        <param name="initial_position" value="0.0"/>
    </node>
</launch>
